frontend/src/components/loginOutButton/index.tsx

```diff
@@ -25,6 +31,14 @@ const LoginOutButton = () => {
         onClick={handleLoginOutButtonClick}>
         {isAuthenticated && !loading ? 'Log out' : 'Log in'}
       </Button>
+      <Button
+        variant="contained"
+        onClick={async () => {
+          const token = await getTokenSilently();
+          console.log(token);
```

Comment:
This `console.log` needs to go.

Reply:
This is here for developers to easily get their access token for testing, it will most definitely be taken out in the future.

I can put some comments around it to make its purpose more obvious?

Reply:
We definitely don't want to forget removing this later, printing sensitive information to the console makes me :scream: .
I suggest removing this now and if someone needs to debug a token later, they'll be able to figure it out.
Alternatively, you can comment it out and add a comment why it's there.

Reply:
I moved it to a separate component in a Dev folder to take it out of any components which will be used in MVP. I'll remove it from the sidebar and then someone can add it in if they wish to use it.